cfgov/search/dotgov.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from django.conf import settings
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+SEARCH_DOT_GOV_AFFILIATE = settings.SEARCH_DOT_GOV_AFFILIATE
+SEARCH_DOT_GOV_ACCESS_KEY = settings.SEARCH_DOT_GOV_ACCESS_KEY
+
+
+def search(query, limit=20, offset=0,
+           enable_highlighting=True, sort_by='relevance'):
+
+    if 1 > limit or limit > 50:
+        raise ValueError('limit has to be between 1-50')
+
+    if 0 > offset or offset > 999:
+        raise ValueError('limit has to be between 1-50')
```

Comment:
should error msg refer to offset and not limit?

Reply:
Yup it should.